### ISSUE TITLE
Wrong MAE calculation

### ISSUE THREAD

**Issue by Atik16209** [shamangary/FSA-Net#17]
First of all thank you for uploading this paper and all the code:) I believe that MAE obtained from running the test and MAE stated in the paper are different. In the section 3.1 of the paper you define your error as average L1 norm of  deviations, but in the  FSANET_test.py (line 429) you calculate average error on each axis and then average the obtained 3 numbers. This is actually 3 times less then it would have been if you calculated according to 3.1 It would be nice if you state this somewhere

**Comment by shamangary** [comment]
Hello. Section 3.1 is referring to the optimization and the general idea  https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/losses.py#L768
while the testing table states that yaw, pitch, roll individually and the testing MAE is computed as the average of the 3 of them indeed. We appreciate your feedback and will add some explanation in the code.
